components/resources/request_otr_strings.grdp

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <grit-part>
   <message name="IDS_REQUEST_OTR_TITLE" desc="Title of request OTR interstitial page.">
```

Comment:
This definitely does not seem like the right fix here. It doesn't make sense to have an empty string for `IDS_REQUEST_OTR_TITLE`, we would just get rid of it entirely, but I'm not sure how this changes the behavior.

Reply:
also this needs tests